server/scanner/Scanner.js

```diff
@@ -477,7 +477,7 @@ class Scanner {
       })
       if (newSeries.length) {
         await this.db.insertEntities('series', newSeries)
-        SocketAuthority.emitter('series_added', newSeries.map(se => se.toJSON()))
+        SocketAuthority.emitter('multiple_series_added', newSeries.map(se => se.toJSON()))
```

Comment:
Added `multiple_series_added` event to distinguish from a single series being added.